packages/flutter_tools/test/flutter_manifest_test.dart

```diff
@@ -0,0 +1,376 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_tools/src/cache.dart';
+import 'package:flutter_tools/src/flutter_manifest.dart';
+
+import 'package:test/test.dart';
+
+import 'src/common.dart';
+import 'src/context.dart';
+
+void main() {
+  setUpAll(() {
+    Cache.flutterRoot = getFlutterRoot();
+  });
+
+  group('FlutterManifest', () {
+    test('has no fonts or assets when the "flutter" section is empty', () async {
+      final String manifest = '''
+name: test
+dependencies:
+  flutter:
+    sdk: flutter
+''';
+      final FlutterManifest flutterManifest = await FlutterManifest.createManifestFromString(manifest);
+      expect(flutterManifest, isNotNull);
+      expect(flutterManifest.appName, 'test');
+      expect(flutterManifest.usesMaterialDesign, false);
+      expect(flutterManifest.fontsDescriptor, isEmpty);
+      expect(flutterManifest.fonts, isEmpty);
+      expect(flutterManifest.assets, isEmpty);
+    });
+
+    test('usesMaterialDesign flag is set', () async {
```

Comment:
Prefixing the group name, the name of this test reads "FlutterManifest usesMaterialDesign flag is set". I'd rename the test to `knows if material design is used` or similar to get "FlutterManifest knows if material design is used".